e2e/tests/image-list.test.js

```diff
@@ -16,13 +12,79 @@ const config = `
 
 const data = {
   images: [
-    'https://data.heartex.net/open-images/train_0/mini/00133643bbf063a9.jpg',
+    'https://data.heartex.net/open-images/train_0/mini/0030019819f25b28.jpg',
     'https://data.heartex.net/open-images/train_0/mini/00155094b7acc33b.jpg',
-    'https://data.heartex.net/open-images/train_0/mini/000e842c55ab7d14.jpg',
-    'https://data.heartex.net/open-images/train_0/mini/00766c7816e51125.jpg',
+    'https://data.heartex.net/open-images/train_0/mini/00133643bbf063a9.jpg',
+    'https://data.heartex.net/open-images/train_0/mini/0061ec6e9576b520.jpg',
   ],
 };
 
+const result = [
+  {
+    'original_width': 768,
+    'original_height': 576,
+    'image_rotation': 0,
+    'value': {
+      'x': 7.814060788954878,
+      'y': 42.05253910374108,
+      'width': 11.226011120078905,
+      'height': 10.491211550533542,
+      'rotation': 0,
+      'rectanglelabels': [
+        'Planet',
+      ],
+    },
+    'id': 'SBHuSbuOoI',
+    'from_name': 'tag',
+    'to_name': 'img',
+    'type': 'rectanglelabels',
+    'origin': 'manual',
+    'item_index': 0,
+  },
+  {
+    'id': 'spPCrj0omt',
+    'type': 'rectanglelabels',
+    'value': {
+      'x': 40.237685381355924,
+      'y': 22.88135593220339,
+      'width': 7.878707627118645,
+      'height': 18.64406779661017,
+      'rotation': 0,
+      'rectanglelabels': [
+        'Moonwalker',
+      ],
+    },
+    'origin': 'manual',
+    'to_name': 'img',
+    'from_name': 'tag',
+    'item_index': 1,
```

Comment:
I think for correct test there is should be more than 4 images (because displayed one + 3 next are preloaded) and at least one index should be 4+

Reply:
yeah, I will add it separately. for now it covers the case. on the other hand, all cases with preloading enabled must be tested